docs/guide/topics/threads.rst

```diff
@@ -0,0 +1,245 @@
+
+.. code-block:: python3
+    :emphasize-lines: 3
+
+    @bot.command()
+    async def create_thread(ctx: commands.Context, *, thread_name: str):
+        thread = await ctx.message.create_thread(name=thread_name)
+
+        await thread.send(f"Hello from the {thread_name} thread!")
+
+and then we have our public thread:-
+
+.. image:: /images/guide/threads/create_public_thread_param.png
+
+As discussed before, we can see it from the thread list, even if we're not a member:-
+
+.. image:: /images/guide/threads/not_a_member.png
+    :scale: 90
+
+
+Private threads
+~~~~~~~~~~~~~~~~
+
+Private threads exist as "invite only" threads.
+By default, they only contain the person who created them, but other people can be invited to participate.
+
+Private threads can optionally be 'invitable'.
+If a thread is invitable, it means anyone in that thread can invite others to it.
+Otherwise, only the owner and people with the :attr:`~Permissions.manage_threads` permission can add new people to the thread.
+
+Users with :attr:`~Permissions.manage_threads` ("Moderators") can also see a list of private threads they're not in, similar to public threads.
+
+Let's create a private, non-invitable thread with a command:
+
+.. code-block:: python3
+    :emphasize-lines: 5
+
+    @bot.command()
+    async def secret_thread(ctx: commands.Context):
+        thread = await ctx.channel.create_thread(
+            name="Secret Birthday Planning!",
+            invitable=False
+        )
+        await thread.send(ctx.author.mention)
+
+We've chosen here to mention the author as soon as we make the thread.
+If we didn't do this, then the thread would be created, but we wouldn't necessarily be able to see it!
+
+Note that this mention is only guaranteed to work because we own the thread.
+Mentions from regular users wouldn't work in this thread, because it is not ``invitable``.
+
+You can also use the :meth:`~Thread.add_user` method to add someone, without sending a mention. This requires ``invitable`` to be ``False``.
+
+Unlike a public thread, private threads are not tied to messages, so they won't appear in the channel they're attached to.
+The member list will look like this inside in the thread:
+
+.. image:: /images/guide/threads/create_private_thread.png
+    :scale: 90
+
+
+.. warning::
+    Thread creation shows up in the audit logs, even if the thread is private.
+    Be wary of this if you don't intend for others to know about the thread.
+
+
+Adding members to threads
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We touched on it earlier a bit, but now we'll go into how to add people to threads.
+
+The go-to practice for doing this is pinging the member, like so:
+
+.. image:: /images/guide/threads/mention_member_to_thread.png
+
+The API has methods of adding members that do not require pinging them, which is implemented with :meth:`Thread.add_user`.
```

Comment:
Again kind of misleading wording since it still looks like a ping to the invited person. 


![image](https://github.com/Rapptz/discord.py/assets/16943959/2bec961f-ffbe-4f11-a9a2-7299c0ff7acf)
![image](https://github.com/Rapptz/discord.py/assets/16943959/49cb2b19-24da-4c00-9395-72a35ae05beb)